samples/tools/testbed/scenarios/human_eval_two_agents.py

```diff
@@ -47,6 +47,7 @@ def run_tests(candidate):
     llm_config={
         # "request_timeout": 180,  # Remove for autogen version >= 0.2, and OpenAI version >= 1.0
         "config_list": config_list,
+        "use_cache": False,
```

Comment:
This kwarg is only available for pyautogen<0.2. Please check #599 for the migration guide.

Reply:
Yes, the code in main runs on autogen 0.1.14 and 0.2.0. I forgot that use_cache was deprecated. I'll migrate over to setting the seed to None and hold this PR until 0.2 is out.